tools/getopts_long.bash

```diff
@@ -0,0 +1,76 @@
+# https://github.com/UrsaDK/getopts_long
```

Comment:
It's MIT which means you also need to add copyright and license itself I believe 🤔

Reply:
Removed